service-registry/src/main/java/io/servicecomb/serviceregistry/client/IpPortManager.java

```diff
@@ -46,10 +47,10 @@
 
   private ArrayList<IpPort> defaultIpPort;
 
-  private InstanceCache instanceCache = null;
-
   private AtomicInteger currentAvailableIndex;
 
+  private boolean autoDiscoveryInited = false;
```

Comment:
The default value is false, we don't need assign the false value here.

Reply:
I think it's better to give explicit default value here to false, because it's more readable.